boot/freeldr/freeldr/lib/mm/heap.c

```diff
@@ -24,6 +24,13 @@ DBG_DEFAULT_CHANNEL(HEAP);
 
 #define FREELDR_HEAP_VERIFIER
 
+#define TAG_FREELDR_COMMONBUFFER    'bcLF'
+#define TAG_FREELDR_DISKCONTEXT     'cdLF'
+#define TAG_FREELDR_MBR             'bmLF'
+#define TAG_FREELDR_PARTITIONS      ' pLF'
+#define TAG_FREELDR_SECTOR          ' sLF'
+#define TAG_FREELDR_SRB             'rsLF'
```

Comment:
Suggestion: Since these tag names are already quite long, could you rename them instead like: `TAG_FLDR_...` ?
(and "COMMONBUFFER" --> "CMNBUF" ; "DISKCONTEXT" --> "DISKCTX" , "PARTITIONS" --> "PART" )

Reply:
Done.